src/views/notifications/Create.vue

```diff
@@ -0,0 +1,323 @@
+                          </div>
+                        </div>
+                        <div class="col-span-12 xxl:col-span-12 mb-4">
+                          <div>
+                            <label for="create-title-notification" class="form-label">
+                              Content
+                            </label>
+                            <textarea
+                              id="create-content-notification"
+                              :class="'form-control' + (this.validation_error?.content != null ? ' border-theme-6' : '')"
+                              placeholder="Enter Content"
+                              rows="5"
+                              v-model="this.create_notification.content"
+                            />
+                            <div v-if="this.validation_error?.content != null" class="text-theme-6 mt-2 mb-4">
+                              {{ this.validation_error?.content[0] }}
+                            </div>
+                          </div>
+                        </div>
+                        <div class="col-span-12 xxl:col-span-6 mb-4">
+                          <div>
+                            <label for="create-color-notification" class="form-label">
+                              Color
+                            </label>
+                            <input
+                              id="create-color-notification"
+                              type="color"
+                              :class="'form-control' + (this.validation_error?.color != null ? ' border-theme-6' : '')"
+                              placeholder="Enter notification reason"
+                              v-model="this.create_notification.color"
+                            />
+                            <div v-if="this.validation_error?.color != null" class="text-theme-6 mt-2 mb-4">
+                              {{ this.validation_error?.color[0] }}
+                            </div>
+                          </div>
+                        </div>
+                        <div class="col-span-12 xxl:col-span-6 mb-4">
+                          <div>
+                            <label class="form-label">
+                              Type
+                            </label>
+                            <TailSelect
+                              v-model="this.create_notification.type"
+                              :class="'form-control' + (this.validation_error?.type != null ? ' border-theme-6' : '')"
+                              :options="{
+                                search: true,
+                                classNames: 'w-full'
+                              }"
+                            >
+                              <option value=1>Icon</option>
+                              <option value=2>User</option>
+                            </TailSelect>
+                            <div v-if="this.validation_error?.type != null" class="text-theme-6 mt-2 mb-4">
```

Comment:
Unnecessary `!= null`

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/34839e538cf6eea37658b6eebcf618108f1b8fc8